packages/graphback-cli/src/index.ts

```diff
@@ -10,6 +11,8 @@ if (require.main === module) {
   // eslint-disable-next-line no-unused-expressions
   yargs
     .commandDir('commands')
+    .command(serve)
+    .command(printSchema)
```

Comment:
I only added the commands from graphql-serve package, Is this okay?

Reply:
This is a very bad pattern. See how commands are created here - we use folders. 
As we have mentioned before we should use the same patterns for all commands and simply wrap them.

Reply:
TL;DR - let's create commands the same way they are created in this package